src/main/resources/htmlpublisher/HtmlPublisherTarget/help-useWrapperFileDirectly.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  By default, each archived reports access generates a new wrapper file.
+  If checked, the legacy wrapper file will be used instead of generate it each time.
+  Must only be used if wrapper file was been transformed and saved with checksum
```

Comment:
What is a “legacy wrapper file”? The term _wrapper_ is not mentioned in https://github.com/jenkinsci/htmlpublisher-plugin/blob/8a37a361a73c34b0ef7de9720b934d03bc0b47a9/README.md at all. As a potential user of the plugin I would have no idea what to make of this help text or whether I should check this box or not.